1727059891159	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1727061926450	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1727061926451	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1727061926451	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1727061926453	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1727061926459	Sync.SyncScheduler	DEBUG	Woke from sleep.
1727061926459	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1727061926459	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1727061926459	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1727061928460	Sync.Service	DEBUG	User-Agent: Firefox/130.0 (Linux x86_64) FxSync/1.132.0.20240829075237.desktop
1727061928460	Sync.Service	INFO	Starting sync at 2024-09-23 08:55:28 in browser session lbeyuhjJFv7X
1727061928461	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1727061928463	Sync.Status	INFO	Resetting Status.
1727061928463	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1727061928464	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1727061928464	FirefoxAccounts	DEBUG	getOAuthToken enter
1727061928464	FirefoxAccounts	TRACE	getCachedToken returning cached token
1727061928464	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1727061928464	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1727061928465	Sync.SyncAuthManager	INFO	Getting sync key
1727061928466	FirefoxAccounts	DEBUG	getOAuthToken enter
1727061928466	FirefoxAccounts	TRACE	getCachedToken returning cached token
1727061928466	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1727061928466	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1727061928466	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1727061928466	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1727061928466	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1727061928468	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1727061928468	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1727061928468	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1727061928468	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1727061928469	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1727061928469	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1727061928469	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1727061928469	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1727061928470	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1727061928470	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1727061928472	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/213784407/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1727061928472	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1727061928472	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1727061928473	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1727061928473	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1727061928473	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1727061928475	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
